components/landing/TheLearnSection/LearnSectionCard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <article class="learn-card">
-    <div class="bx--col-max-3 bx--col-lg-4 bx--col-md-3 learn-card__content">
+    <div class="cds--col-max-3 cds--col-lg-4 cds--col-md-3 learn-card__content">
       <div
         class="bx--aspect-ratio bx--aspect-ratio--4x3 learn-card__content__aspect-ratio"
```

Comment:
non-blocking comment:

in the future, this will probably have to update to `cds--`

Reply:
Thank you! Changed in https://github.com/Qiskit/qiskit.org/pull/2986/commits/ce723d986cbe10f4244189e41d9cb2597faade80.